README.md

```diff
@@ -21,7 +21,14 @@ This README documents the current strawman architecture.  Changes must be made v
     2. ARAs respond to ARS with Message
     3. ARA sends query messages to KPs
     4. KPs respond to ARAs with Message
-3. All communication between components conforms to the ReasonerAPI Message spec
+3. Interfaces:
+    1. All communication between the ARS and ARAs conforms to the ReasonerAPI Message spec
+    2. KP can expose their information using these methods:
+       A. ReasonerAPI Message
+       B. Any SmartAPI-annotated interface
```

Comment:
My understanding is that, currently, conformance with [the SmartAPI specification](https://github.com/SmartAPI/smartAPI-Specification/blob/OpenAPI.next/versions/3.0.0.md) alone is not sufficient for Translator compatibility, but that a particular extension is required as well. @newgene may be able to help.

Reply:
@patrickkwang  The current SmartAPI spec in the repo you linked is a little out-dated. A new version is available [here](https://x-bte-extension.readthedocs.io/en/latest/), and we will update the SmartAPI spec repo soon. Meanwhile, we are also actively developing a Translator landing page within SmartAPI, specifically discussing the best practice to create an KP API for translator and how to annotate it with SmartAPI, and will share with the translator community soon.

Reply:
So ca we tighten up the language here a little bit?  "Translator-style SmartAPI-annotated?" or "biolink-compliant SmartAPI-annotated?" with specification to be filled in later?

Reply:
These two things look like very different sorts of things, rather than versions of the same thing, but presumably this will become clearer once the documentation is updated.

Reply:
Agreed on the value of making that a bit more precise.  We propose "SmartAPI (semantically annotated API)" to describe the SmartAPI records with sufficient semantic annotation to map to BioLink Model.

Reply:
Right, let's label this as "SmartAPI (semantically annotated API)". When referencing this approach, we should just call it SmartAPI. We are updating the SmartAPI website to make it more clear and consistent. @patrickkwang @cbizon